inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -468,8 +468,15 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     }
 
     // Try to create operation from loaded opsets
-    if (!ngraphNode && opsets.count(params.version)) {
-        auto opset = opsets.at(params.version);
+    GenericLayerParams params_copy = params;
```

Comment:
Why do we need to copy all parameters? I think we can create a local variable which contains opset name and change if it is needed.

Reply:
Done.